ui/config-overrides.js

```diff
@@ -17,16 +17,31 @@ const WebpackBar = require('webpackbar')
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 const rewireHtmlWebpackPlugin = require('react-app-rewire-html-webpack-plugin')
 
+function copyDistroRes() {
+  const distroResPath = '../bin/distro-res'
+  if (fs.existsSync(distroResPath)) {
+    fs.copySync(distroResPath, './public/distro-res')
```

Comment:
Copy files to the public folder cause unnecessary git changes, this makes the solution not perfect, but I can't find a way to resolve it. But I think it is not that important because this logic only happens when you debug this feature.

Reply:
Looks fine to me, since there will be git changes from the very beginning implementation of @shhdgit .